src/cms/templates/users/welcome_email.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+{% autoescape off %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body>
+    <div style="text-align:justify; font-family:sans-serif; background-color: rgba(241, 245, 249, 100); padding: 1rem; border-radius: 1rem; line-height:1.5">
+        <img src='cid:logo' style="width:200px;"> 
+        <br>
+        {% trans 'Welcome' %} {{ user.profile.full_user_name }},
+        <br> 
+        <br> 
+        {% trans 'You are receiving this e-mail, because an Integreat-account was created for you.' %}
+        {% if uid and token %}
+            {% trans 'Please click the link below to set a password and activate your account.' %}
+            <a href={{ base_url }}{% url 'activate_account' uidb64=uid token=token %} 
+            style="color:rgb(0, 0, 200);"> {{ base_url }}{% url 'activate_account' uidb64=uid token=token %}  </a>
+        {% else %}
+            {% trans 'Please click the link below to log in with your password and to your account.' %}
+            <a href={{ base_url }} style="color:rgb(0, 0, 200);"> {{ base_url }} </a>
+        {% endif %}
+        <br>
+        <br>
```

Comment:
```suggestion
            <br>
            <br>
            <a href={{ base_url }}{% url 'activate_account' uidb64=uid token=token %} 
            style="color:rgb(0, 0, 200);"> {{ base_url }}{% url 'activate_account' uidb64=uid token=token %}  </a>
        {% else %}
            {% trans 'Please click the link below to log in with your password and to your account.' %}
            <br>
            <br>
            <a href={{ base_url }} style="color:rgb(0, 0, 200);"> {{ base_url }} </a>
        {% endif %}
```
I think these newlines are better suited between the text and the link like they were before.